CHANGELOG.md

```diff
@@ -5,7 +5,9 @@
     Click to see more.
   </summary>
 
-  - This is a change log example item
+  - Minor: Create linter for new MDS colors
```

Comment:
I like this new approach! Calling out the severity of each change is a great idea.